db/seeds.d/100-installation_media.rb

```diff
@@ -10,11 +10,6 @@
       :os_family => "Redhat",
       :path => "http://mirror.centos.org/centos/$major/os/$arch",
     },
-    {
-      :name => "CentOS 8 mirror",
-      :os_family => "Redhat",
-      :path => "http://mirror.centos.org/centos/$major/BaseOS/$arch/kickstart",
-    },
     {
       :name => "CentOS Stream",
```

Comment:
This will can be dropped once CentOS Stream 8 goes EOL next year. mirror.centos.org will disappear and the CentOS Stream 9 mirror becomes the regular CentOS Stream mirror. I assume version 10 will also appear there.

Perhaps it's something we should already prepare for: rename this to `CentOS Stream 8 mirror` and rename `CentOS Stream 9 mirror` to `CentOS Stream mirror`.